poly-commit/src/data_structures.rs

```diff
@@ -178,7 +179,7 @@ impl<'a, F: Field, P: Polynomial<F>> LabeledPolynomial<F, P> {
 }
 
 /// A commitment along with information about its degree bound (if any).
-#[derive(Clone)]
```

Comment:
@Pratyush If I'm not deriving `Absorb`, what about the `degree_bound`? I would assume this doesn't need to get absorbed.